Client/Frontend/Browser/BraveWebView.swift

```diff
@@ -7,6 +7,8 @@ import WebKit
 
 class BraveWebView: WKWebView {
     
+    var isSecure = false
```

Comment:
Seems a bit redundant/problematic to track the secure-ness with this boolean when we just call the `updateURLBar(forTab:)` anyways. Could we not just determine secure-ness in that call (since it would be called when both `hasOnlySecureContent` and `serverTrust` are changed?)

Reply:
Unfortunately we cannot determine secure-ness in that call, due to when changing tabs as we need to remember the secureness of the web view.

Reply:
Hmm, then is it possible then to make this a computed read-only property? I know that one of those call's is async, so if it's possible then I will accept this solution and mark as approved :)

Reply:
Thanks, if you could approve and merge that would be great as it will unblock my next ticket